docs/sources/reference/commandline/cli.md

```diff
@@ -1501,6 +1504,7 @@ removed before the image is removed.
       --ipc=""                   Default is to create a private IPC namespace (POSIX SysV IPC) for the container
                                    'container:<name|id>': reuses another container shared memory, semaphores and message queues
                                    'host': use the host shared memory,semaphores and message queues inside the container.  Note: the host mode gives the container full access to local shared memory and is therefore considered insecure.
+      -l, --label=[]             Set labels
```

Comment:
Should probably explain what labels are. E.g. "Set key/value metadata on container"